presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleRecordCursor.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.plugin.blackhole;
+
+import com.facebook.presto.spi.RecordCursor;
+import com.facebook.presto.spi.type.Type;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+
+import java.util.List;
+
+import static java.lang.Integer.max;
+
+public class BlackHoleRecordCursor
+        implements RecordCursor
+{
+    private static final byte[] ZERO_BYTES = new byte[] {42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42};
```

Comment:
I don't like 42, I would prefer to have something more human readable. Maybe read value of bytes from session variable?